proof-producer/bin/proof-producer/CMakeLists.txt

```diff
@@ -65,6 +68,11 @@ function(setup_proof_generator_target)
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
     )
 
+    target_include_directories(${ARG_TARGET_NAME}-lib INTERFACE
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
```

Comment:
Please add also `$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>`. Now the list of headers for this target is empty in export file, and it works only because dependent targets also have headers in this directory.

Reply:
done